### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by leahpar47**

### CODE REVIEW

**Comment by robi-y** [comment]
Misses some of the requirement for the introduction part
see: https://github.com/jce-il/sw-modeling-2017b/blob/master/Homework.md#stage-1---introducion
(please comment below if you add stuff)

File: React/README.md [hunk 1]
```diff
@@ -10,4 +10,79 @@ React is a JavaScript library for building user interfaces.
...
+React is an open-source Javascript library, firstly develop by Facebook and used by a lot of very big companies (like Airbnb, Reddit,  SalesForce, Twitter, WhatsApp and of course Facebook and Instagram). React processes only user interfaces in applications. This is the View in the MVC (Model-View-Controller) template.
+
+React was created by Jordan Wlake, a software enginner at Facebook. He was influenced by XHP, an HTML component framework for PHP. It was first deployed on Facebook's newsfeed in 2011 and later on Instagram.com in 2012.It was open-sourced at JSConf US in May 2013. 
+
+The React Homepage is hosted here (https://facebook.github.io/react/) and the repository can be find at :
+https://github.com/facebook/react/
+
+### Why it was chosen ?
+
+The is one of the hotest JS library at this moment. Used by a lot a developers all around the world (and Ilya using it every day at work).
```
Comment (robi-y) [inline React/README.md]: This is..

File: React/README.md [hunk 2]
```diff
@@ -10,4 +10,79 @@ React is a JavaScript library for building user interfaces.
...
+![Pr](pullrequest.png)
+
+[Commits](https://github.com/facebook/react/commits/master)
+![Commit Graph](graphcommit.png)
+
+[Contributors](https://github.com/facebook/react/graphs/contributors)
+![Top 6 contributors](top6contributor.png)
+
+
+Start as an internal Facebook projet but now mainteant by a huge community and React is worked on full-time by Facebook's product infrastructure and Instagram's user interface engineering teams. They're often around and available for questions on :
```
Comment (robi-y) [inline React/README.md]: improve English

**Comment by robi-y** [comment]
Hi,
A good start
Added some comments inside
There are quite a few English problems.
We need some more info on the development process - not only about submitting issues.
And resources about modeling in the project.

**Comment by robi-y** [comment]
Any news? Would you like to present tomorrow?

**Comment by ZakiNata** [comment]
Hi @robi-y , 

since we are not managing to find a suitable time, can you check our work so far and review it ? afterwards we can make more adjustments if needed 

Thanks,
Zaki

**Pull request merged**